test/sharness/t0500-issues-and-regressions-offline.sh

```diff
@@ -8,17 +8,17 @@ test_description="Tests for various fixed issues and regressions."
 
 test_expect_success "ipfs init with occupied input works - #2748" '
 	export IPFS_PATH="ipfs_path"
-	echo "" | time-out ipfs init &&
+	echo "" | time-out 10 ipfs init &&
 	rm -rf ipfs_path
 '
 test_init_ipfs
 
 test_expect_success "ipfs cat --help succeeds with no input" '
```

Comment:
the text here is not right, it should be `"ipfs cat --help succeeds even if stdin remains open"` 

correct me if i'm wrong, but afaiu (from chatting with @whyrusleeping) this is the case.